### ISSUE TITLE
why model compiling?

### ISSUE THREAD

**Issue by xprabhudayal** [SakanaAI/AI-Scientist#91]
![image](https://github.com/user-attachments/assets/cf341142-b110-4e88-a232-3da750090021)
does there any usage of the models (nanogpt, nanogpt_lite ...) in later time in `launch_scientist.py` or is it just for comparing with the baseline results?

in this image we are compiling the model, & upon later analysis or the code in `launch_scientist.py` 
we are just comparing with the baseline results

**Comment by conglu1997** [comment]
Yes these are the baseline runs to compare any future ideas or proposals against.

**Comment by xprabhudayal** [comment]
Can't we just distribute the cached baseline results, to avoid  the heavy
task of compiling 🤔

**Comment by conglu1997** [comment]
See the FAQ, no because things like runtime comparisons are machine dependent :)

**Comment by xprabhudayal** [comment]
@conglu1997 i've been using cloud instance to run this ai scientist, it works fine. 
i just wanna know that if all these work in COLAB PRO with A100? as many of the code were TERMINAL based../?
